monitoring/prober/test_isa_simple.py

```diff
@@ -107,6 +112,24 @@ def test_get_isa_by_search_latest_time_excluded(session, isa1_uuid):
   assert isa1_uuid not in [x['id'] for x in resp.json()['service_areas']]
 
 
+def test_get_isa_by_search_area_only(session, isa1_uuid):
+  resp = session.get('/identification_service_areas'
+                     '?area={}'.format(common.GEO_POLYGON_STRING))
+  assert resp.status_code == 200
+  assert isa1_uuid in [x['id'] for x in resp.json()['service_areas']]
+
+
+def test_get_isa_by_search_huge_area(session, isa1_uuid):
+  resp = session.get('/identification_service_areas'
+                     '?area={}'.format(common.HUGE_GEO_POLYGON_STRING))
+  assert resp.status_code == 413
```

Comment:
If this test passes currently, we should be suspicious of something -- HUGE_GEO_POLYGON_STRING has latitudes greater than 90 degrees, so we should return 400 before HUGE_VERTICES is fixed.

Reply:
This test sort of pass in a way. It returns the correct message indicating the area is too large. But the code returned is incorrect. So that means the server knows how to respond but just uses the wrong code

Reply:
Yeah, there are two things wrong here.  First, lat=130 (invalid area) should return 400, not 413.  Second, valid areas that are too large should return 413, not 400.  In a different PR, we should add a test with lat=130 to reveal the additional flaw.